httpclient/config.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright © 2024 Acronis International GmbH.
+
+Released under MIT license.
+*/
+
+package httpclient
+
+import (
+	"github.com/acronis/go-appkit/config"
+	"time"
+)
+
+const (
+	// DefaultLimit is the default limit for rate limiting.
+	DefaultLimit = 10
```

Comment:
The name DefaultLimit seems too generic for a package-level constant.

Additionally, I’m not convinced that default values are necessary in this case. Please refer to the second part of my [next comment](https://github.com/acronis/go-appkit/pull/18/files#r1893834939) for further details.